src/main/groovy/com/calgaryscientific/gradle/VeracodeUploadFileTask.groovy

```diff
@@ -78,10 +67,10 @@ class VeracodeUploadFileTask extends VeracodeTask {
     }
 
     void run() {
-        UploadAPIWrapper update = uploadAPI()
         def error
         Integer tries = 1;
         Integer maxTries = Integer.parseInt((getMaxUploadAttempts() != null) ? getMaxUploadAttempts() : '10')
+        Integer waitTime = Integer.parseInt((this.waitTimeBetweenAttempts != null) ? this.waitTimeBetweenAttempts : '5000')
 
         println ''
         if (tries > 1) {
```

Comment:
I would extract the entire while(....) loop to separate function and name it uploadFile so the flow would be
```
for each file -> uploadFile
```

- the uploadFile would throw exception or log warnings
- as the first step, I'm not sure if we want to print the logs to the output file or just use log.error/info/debug to the console and it's good enough?